honeybadgermpc/progs/mimc_jubjub_pkc.py

```diff
@@ -0,0 +1,77 @@
+import asyncio
+from random import randint
+from honeybadgermpc.field import GF
+from honeybadgermpc.mpc import PreProcessedElements, Subgroup
+from honeybadgermpc.elliptic_curve import Point, Jubjub
+from honeybadgermpc.progs.jubjub import share_mul
+from honeybadgermpc.progs.mimc import mimc_mpc, mimc_plain
+
+pp_elements = PreProcessedElements()
```

Comment:
actually, that could potentially make this have way worse performance as it would have to read the preprocessing file each time you run the method -- keep this variable like this.